src/http/api/views/account.follows.viewer.integration.test.ts

```diff
@@ -0,0 +1,434 @@
+import { AsyncEvents } from 'core/events';
+import type { Knex } from 'knex';
+import { generateTestCryptoKeyPair } from 'test/crypto-key-pair';
+import { createTestDb } from 'test/db';
+import {
+    afterEach,
+    beforeAll,
+    beforeEach,
+    describe,
+    expect,
+    it,
+    vi,
+} from 'vitest';
+import { KnexAccountRepository } from '../../../account/account.repository.knex';
+import { AccountService } from '../../../account/account.service';
+import type { InternalAccountData, Site } from '../../../account/types';
+import { FedifyContextFactory } from '../../../activitypub/fedify-context.factory';
+import { AccountFollowsViewer } from './account.follows.viewer';
+
+vi.mock('@fedify/fedify', async () => {
+    const actual = await vi.importActual('@fedify/fedify');
+    return {
+        ...actual,
+    };
+});
+
+describe('AccountFollowsViewer', () => {
+    let viewer: AccountFollowsViewer;
+    let accountService: AccountService;
+    let events: AsyncEvents;
+    let site: Site;
+    let internalAccountData: InternalAccountData;
+    let db: Knex;
+
+    beforeAll(async () => {
+        db = await createTestDb();
+    });
+
+    afterEach(() => {
+        events.removeAllListeners();
```

Comment:
I don't think we need this - `events` will just fall out of scope and be cleaned up